CentOS-6-32/Dockerfile

```diff
@@ -22,6 +21,12 @@ RUN yum -y update; yum -y install \
                    tar \
                    zlib-devel.i686
 RUN yum clean all
+RUN curl -fSL "https://cmake.org/files/v3.1/cmake-3.1.3-Linux-i386.sh" -o /usr/cmake.sh
+RUN cd /usr \
+ && chmod +x ./cmake.sh
+
+RUN cd /usr \
+&& ./cmake.sh --skip-license
```

Comment:
To reduce the size of the image, you can use:
```dockerfile
RUN curl -fSL "https://cmake.org/files/v3.1/cmake-3.1.3-Linux-i386.sh" | bash -s --skip-license
```
That avoids to have the temporary file `/usr/cmake.sh` in the image.